app/main.py

```diff
@@ -2,0 +1,16 @@
+from app.cafe import Cafe
+from app.errors import NotVaccinatedError, NotWearingMaskError
+
+
+def go_to_cafe(self, friends: list[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            cafe.visit_cafe(visitor)
+            return f"Friends can go to {cafe.name}"
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except NotVaccinatedError:
```

Comment:
The function should also handle the `OutdatedVaccineError` exception, as it is defined in the project. Consider adding an exception handler for it to provide a more comprehensive error handling.